config.go

```diff
@@ -91,9 +89,10 @@ const (
 
 // ConfigStruct is a structure holding the whole service configuration
 type ConfigStruct struct {
-	Storage StorageConfiguration `mapstructure:"storage" toml:"storage"`
-	Logging LoggingConfiguration `mapstructure:"logging" toml:"logging"`
-	Cleaner CleanerConfiguration `mapstructure:"cleaner" toml:"cleaner"`
+	Storage StorageConfiguration              `mapstructure:"storage" toml:"storage"`
+	Logging logger.LoggingConfiguration       `mapstructure:"logging" toml:"logging"`
+	Cleaner CleanerConfiguration              `mapstructure:"cleaner" toml:"cleaner"`
+	Sentry  logger.SentryLoggingConfiguration `mapstructure:"sentry" toml:"sentry"`
 }
 
 // LoggingConfiguration represents configuration for logging in general
```

Comment:
shouldn't you drop this configuration as you are now using `logger.LoggingConfiguration`